sockets/1.2/modules/ROOT/pages/sockets-connector-studio.adoc

```diff
@@ -0,0 +1,253 @@
+= Use Studio to Configure Sockets Connector 1.2 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure Socket Listener source for the connector's flow>>.
+. <<add-connector-operation, Configure Send and Receive Operations in the flow>>.
+. <<configure-other-fields,Configure Supported TCP Protocols>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:sockets-documentation.adoc[Sockets Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Sockets Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `Sockets` in the search field.
+. Click *Sockets* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure Socket Listener Source
+
+An input source initiates a flow when a specified condition is met. The Socket Listener operation represents a socket endpoint that clients can connect to. By browsing to this endpoint, external clients establish a TCP/UDP Socket connection and trigger the flow each time they send a message.
```

Comment:
I have been using the term "input source" to distinguish the source from the other kinds of operations (even though an input source is an operation strictly speaking). "The Socket Listener input source represents..." Also, in formal English, we don't end with "to." "The Socket Listener input source represents a socket endpoint to which clients can connect.